### ISSUE TITLE
 SuiteSparse::CHOLMOD    but the target was not found

### ISSUE THREAD

**Issue by ioir123ju** [colmap/glomap#101]
I use command:
cmake ..    -GNinja   -DCMAKE_CUDA_ARCHITECTURES=75   -DSuiteSparse_CHOLMOD_LIBRARY="/usr/lib/x86_64-linux-gnu/libcholmod.so"  -DSuiteSparse_CHOLMOD_INCLUDE_DIR="/usr/include/suitesparse"
And I'm sure I have /usr/lib/x86_64-linux-gnu/libcholmod.so and  /usr/include/suitesparse.
![image](https://github.com/user-attachments/assets/ea4b0ed2-4a7b-47e2-a9d7-fc71996648e0)

**Comment by ichsan2895** [comment]
How about using this command only?
```
pip install cmake==3.28.4

cmake .. -GNinja -DCMAKE_CUDA_ARCHITECTURES=75
```

I had success for it without explicitly pointing `-DSuiteSparse_CHOLMOD_LIBRARY="/usr/lib/x86_64-linux-gnu/libcholmod.so" -DSuiteSparse_CHOLMOD_INCLUDE_DIR="/usr/include/suitesparse"`

**Comment by ioir123ju** [comment]
Yes, I saw suitesparse in glomap/build/CMakeCache.txt, but "cmake .. -GNinja -DCMAKE_CUDA_ARCHITECTURES=75 "
still reporting same errors. My cmake version is 3.30.1
![image](https://github.com/user-attachments/assets/3bc8afed-2332-4fa6-86fc-672164d52169)

**Comment by ioir123ju** [comment]
> 只使用这个命令怎么样？
> 
> ```
> pip install cmake==3.28.4
> 
> cd path/to/glomap
> 
> mkdir build
> 
> cd build
> 
> cmake .. -GNinja -DCMAKE_CUDA_ARCHITECTURES=75
> ```
> 
> 我没有明确指出就成功了`-DSuiteSparse_CHOLMOD_LIBRARY="/usr/lib/x86_64-linux-gnu/libcholmod.so" -DSuiteSparse_CHOLMOD_INCLUDE_DIR="/usr/include/suitesparse"`
> 
> 它自动在`glomap/build/CMakeCache.txt`
> 
> ```
> //Path to a file.
> SuiteSparse_CHOLMOD_INCLUDE_DIR:PATH=/usr/include/suitesparse
> 
> //Path to a library.
> SuiteSparse_CHOLMOD_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libcholmod.so
> ```

Yes, I saw suitesparse in glomap/build/CMakeCache.txt, but "cmake .. -GNinja -DCMAKE_CUDA_ARCHITECTURES=75 "
still reporting same errors. My cmake version is 3.30.1.
![image](https://github.com/user-attachments/assets/3bc8afed-2332-4fa6-86fc-672164d52169)
